src/maidsafe/vault/maid_manager/action_create_remove_account.h

```diff
@@ -96,9 +96,15 @@ std::string ActionCreateRemoveAccount<Remove>::Serialise() const {
   return action_proto.SerializeAsString();
 }
 
+#ifndef WIN32
```

Comment:
I think we'd voted to use the compiler's native definitions for stuff like this from now on, so `#ifndef _MSC_VER`.  Failing that, we have our own definitions for this: `MAIDSAFE_WIN32`.

Either way, I'm still happy to merge this.  But maybe the next commit to Vaults could change these to `_MSC_VER` as a simple side issue?